src/main/java/io/github/externschool/planner/aop/LoggingAspect.java

```diff
@@ -0,0 +1,75 @@
+package io.github.externschool.planner.aop;
+
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.aspectj.lang.reflect.MethodSignature;
+
+
+import java.util.Arrays;
+
+@Aspect
+@Component
+public class LoggingAspect {
+    private Logger log = LoggerFactory.getLogger(getClass());
+
+    @Pointcut("within(io.github.externschool.planner.controller..*) || within(io.github.externschool.planner.service..*) ")
+    private void allMethods() {
+
+    }
+
+        @Before("allMethods()")
+    public void log(JoinPoint joinPoint) {
```

Comment:
Annotation starts at the same column as the method declaration, please remove extra spaces before '@Before'